gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/task/EhrExtractMessageHandlerTest.java

```diff
@@ -113,7 +114,7 @@ public void When_HandleMessageWithValidDataIsCalled_Expect_CallsAttachmentHandle
 
     @Test
```

Comment:
Do we need a test to confirm an error is thrown under Inline AttachementProcessingException conditions?

Reply:
Yes this test is necessary, I'll add it.